frontend/src/components/molecules/DetailView.tsx

```diff
@@ -23,11 +23,18 @@ const DetailView: React.FC = () => {
     description: '',
     moreInformation: '',
   });
+  const [objectAnnotations, setObjectAnnotations] = useState<Annotation>({
+    label: '',
+    description: '',
+    moreInformation: '',
+  });
+
   const [contributions, setContributions] = useState<Array<Node>>([]);
   const [tradeOffs, setTradeOffs] = useState<Array<Node>>([]);
   const [developmentAreas, setDevelopmentAreas] = useState<Array<Node>>([]);
   const [expanded, setExpanded] = useState<boolean>(false);
   const [selectedConnection, setSelectedConnection] = useState<Node>();
+  const [selectedPredicate, setSelectedPredicate] = useState<Array<string>>(['Loading', 'Loading']);
```

Comment:
```suggestion
  const [selectedPredicate, setSelectedPredicate] = useState<Array<string>>([]);
```

If there is ever a case where this text would not be set, having it set to loading would be misleading. If not, it's fine to have loading as a default value